<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>783</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Kidney Stone Prediction</string>
  </property>
  <property name="windowOpacity">
   <double>100.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: rgb(0, 0, 0)
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>28</width>
    <height>28</height>
   </size>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(167, 191, 232, 1), stop:1 rgba(97, 144, 232, 1));
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <property name="horizontalSpacing">
     <number>15</number>
    </property>
    <property name="verticalSpacing">
     <number>9</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="mainPannel" native="true">
      <property name="styleSheet">
       <string notr="true">background: #ffffff;



 border: 2px solid black;
border-radius: 20px;
color: black;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>18</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>18</number>
       </property>
       <property name="bottomMargin">
        <number>18</number>
       </property>
       <property name="horizontalSpacing">
        <number>13</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="2" column="1" colspan="2">
        <widget class="QLabel" name="canvas">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: black;

color: white;
border: 4px solid;
border-radius:25px;
background-repeat: none;
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,0,0,1), stop:1 rgba(67,67,67,1));</string>
         </property>
         <property name="text">
          <string>Upload CT/MRI images of Kidney to detect...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="head_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background: Transparent;
color: black;
border: 0px;</string>
         </property>
         <property name="text">
          <string>Kidney Stone Prediction</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QWidget" name="sidePannel" native="true">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0,0,0,1), stop:1 rgba(67,67,67,1));


 border: 2px solid black;
border-radius: 20px;
color:white;</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="select_button">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(167, 191, 232, 1), stop:1 rgba(97, 144, 232, 1));




border-radius: 8px;
border:0px;
color:black;</string>
            </property>
            <property name="text">
             <string>Upload</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="detect_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(167, 191, 232, 1), stop:1 rgba(97, 144, 232, 1));

border-radius: 8px;
color:black;border:0px;</string>
            </property>
            <property name="text">
             <string>Predict Image</string>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="result_label">
            <property name="maximumSize">
             <size>
              <width>348</width>
              <height>450</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background:transparent;
border:0px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>8</number>
            </property>
            <property name="indent">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="head">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background: Transparent;
color: white;
border: 0px;</string>
            </property>
            <property name="text">
             <string>Description</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <property name="margin">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
